### PR TITLE
replace httr::POST() with httr::RETRY()

### PR DESCRIPTION
**Pull request by jameslamb**
## Summary

Hey @jdblischak thanks for for this awesome project! I'm working on https://github.com/chircollab/chircollab20/issues/1 as part of Chicago R Collab.

In this PR, I'd like to propose swapping out calls to `httr::POST()` etc. with `httr::RETRY()`. This will make the package more resilient to transient problems like brief network outages or periods where the service(s) it hits are overwhelmed. In my experience, using retry logic almost always improves the user experience with HTTP clients.

Thanks for your time and consideration.

## Checklist

- [x] I agree to follow the [Code of Conduct][conduct]
- [x] I have read the [contributing guidelines][contributing]
- [x] I ran the script [scripts/contribute.R][contribute.R]

[conduct]: https://github.com/jdblischak/workflowr/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/jdblischak/workflowr/blob/master/CONTRIBUTING.md
[contribute.R]: https://github.com/jdblischak/workflowr/blob/master/scripts/contribute.R

### CODE REVIEW

File: R/wflow_use_github.R [hunk 1]
```diff
@@ -134,6 +134,7 @@
 #' }
 #'
 #' @importFrom httpuv startServer
+#' @importFrom httr RETRY
```
Comment (jdblischak) [inline R/wflow_use_github.R]: Please remove the roxygen tag because you explicitly use the namespace `httr::`

File: R/wflow_use_github.R [hunk 2]
```diff
@@ -410,14 +411,30 @@ create_gh_repo <- function(account, repository, account_is_organization = FALSE)
...
+      req_create <- httr::RETRY(
+        verb = "POST"
+        , url = glue::glue("https://api.github.com/orgs/{account}/repos")
+        , config = token
+        , ua
+        , body = list(name = repository)
+        , encode = "json"
+        , terminate_on = c(
+          403, 404
+        )
```
Comment (jdblischak) [inline R/wflow_use_github.R]: This is great! But could you please follow the style of the function call as it was?

**Review by jdblischak — CHANGES_REQUESTED** [review]
This is awesome! Thanks for making workflowr HTTP calls more robust. I made a few minor requests. Once these are updated, I'll be happy to merge your PR.

File: NAMESPACE [hunk 3]
```diff
@@ -40,3 +40,4 @@ export(wflow_use_gitlab)
 export(wflow_view)
 import(rmarkdown)
 importFrom(httpuv,startServer)
+importFrom(httr,RETRY)
```
Comment (jdblischak) [inline NAMESPACE]:
```suggestion
importFrom(httrz,RETRY)
```

File: R/wflow_use_github.R (hunk 1 above)
Comment (jameslamb) [reply R/wflow_use_github.R]: oh yeah, no problem

Comment (jameslamb) [reply R/wflow_use_github.R]: fixed in https://github.com/jdblischak/workflowr/pull/199/commits/99036e145858ea2b9fb66749c571a270a5179e7f

File: NAMESPACE (hunk 3 above)
Comment (jameslamb) [reply NAMESPACE]: this was a teaching  example

File: R/wflow_use_github.R (hunk 2 above)
Comment (jameslamb) [reply R/wflow_use_github.R]: oh sure, sorry! fixed in https://github.com/jdblischak/workflowr/pull/199/commits/99036e145858ea2b9fb66749c571a270a5179e7f

**Review by jdblischak — APPROVED** [review]

**Pull request merged**